### ISSUE TITLE
OSX may break replay

### ISSUE THREAD

**Issue by ChrisKeefe** [ChrisKeefe/provenance_py#70]
When replaying an extracted zip archive of files that was compressed on OSX, parsing fails because the included `__MACOSX` directory contains non-zipfiles named` ._something.qz*`:
![image](https://user-images.githubusercontent.com/39198770/161882564-a73c8a56-b82f-42fd-a202-1e12dbc1ac35.png)

We need to:
- confirm that this only impacts compressed and then decompressed ziparchives
- check whether this behavior also occurs on MacOS
- fix, possibly by ignoring hidden files, ignoring files with `__MACOSX` in the fp, or by catching `BadZipFile("File is not a zip file")` errors and looking more closely at them.

**Comment by ChrisKeefe** [comment]
@cherman2 helped diagnose this issue today. Thank you!

- Provenance Replay assumes that any file suffixed `.qza` or `.qzv` is a QIIME 2 Archive
- the default zip utility used by MacOS finder (`Archive Utility') adds a `__MACOSX` directory when it zips things. Inside this directory are a collection of files named `._<original-archive-name>.qza` that are _not_ QIIME 2 Archives. 
- the `zip` command available from the MacOS terminal does not appear to do this
- when these problematic zip archives are unzipped on MacOS (via terminal and presumably also finder), the resulting decompressed directory does not contain a `__MACOSX` directory`. 
- when unzipped on linux, the false `.qzx` files in the resulting `__MACOSX` directory break parsing with the confusing error message below.

TLDR: this error only arises in cases where a mac user has zipped a collection of `.qzx` and sent it to a non-MacOS machine where it was unzipped for parsing.
